xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -1075,5 +1082,92 @@ absl::StatusOr<std::vector<uint8_t>> CompileToHsaco(
 
 }  // namespace amdgpu
 
+namespace {
```

Comment:
Nit: Please add a blank line after this.

Reply:
Done.